src/Shared/E2ETesting/SauceConnectServer.cs

```diff
@@ -0,0 +1,262 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Concurrent;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Reflection;
+using System.Runtime.InteropServices;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.E2ETesting;
+using Microsoft.Extensions.Internal;
+using Xunit.Abstractions;
+using Xunit.Sdk;
+
+namespace Microsoft.AspNetCore.E2ETesting
+{
+    public class SauceConnectServer : IDisposable
```

Comment:
Most of this file is copy-paste from SeleniumStandaloneServer

Reply:
Is there room to consolidate the two types? It might be difficult to remember to update both types if we change the code in the other.